datalab/kernel/__init__.py

```diff
@@ -31,7 +31,10 @@
 import datalab.bigquery.commands
 import datalab.context.commands
 import datalab.data.commands
-import datalab.mlalpha.commands
+try:
+  import datalab.mlalpha.commands
+except:
+  print('TensorFlow and CloudML SDK are required.')
```

Comment:
Can we elaborate more on this, mainly make it clear what will be disabled as a result. Maybe also adding links to install TF and CloudML would be a good idea, I think a long comment is good here.